pkg/virtctl/vm/remove_volume.go

```diff
@@ -0,0 +1,89 @@
+/*
+ * This file is part of the KubeVirt project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * Copyright 2018 Red Hat, Inc.
+ *
+ */
+
+package vm
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/spf13/cobra"
+	"k8s.io/client-go/tools/clientcmd"
+	v1 "kubevirt.io/api/core/v1"
```

Comment:
I think add and remove volume should be in the same file

Reply:
Do they share any code? If not I would vote for keeping them separate to avoid files becoming unmaintainable again (like `vm.go`).

Reply:
I probably like each command in its own file more, but to be honest I've also thought about grouping at least some commands together (eg. start, stop, restart). But as Felix mentioned they share almost no code, so I put them in their own file.

Reply:
They might not share the same code but they are with the same context of hotplugging and unplugging a volume, so in my thoughts they should be in the same file. Nothing that is not related to hotplug should be added to this file so dont think it will become unmaintainable. you can also see that the UT of add/remove volume are combined so if you would decide to separate the UT they will be in the same file anyways